tools/buildsys/Cargo.toml

```diff
@@ -9,6 +9,7 @@ publish = false
 exclude = ["README.md"]
 
 [dependencies]
+bottlerocket-variant = { version = "0.1", path = "../../sources/bottlerocket-variant" }
```

Comment:
yay. but also weird. do we have tools crates depending on sources crates and sources crates depending on tools crates? should we consider having a `common` directory someday?